<?xml version="1.0" encoding="UTF-8"?>
<!-- Minimal Qt Designer .ui XML for SalesFrame -->
<ui version="4.0">
 <class>SalesFrame</class>
 <widget class="QWidget" name="SalesFrame">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="saleLabel">
     <property name="text">
      <string>Sales</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="saleTable">
     <property name="columnCount">
      <number>4</number>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="addBtnLayout">
     <item>
      <widget class="QPushButton" name="vegBtn">
       <property name="text">
        <string>Veg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="miscBtn">
       <property name="text">
        <string>Misc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="totalLayout">
     <item>
      <widget class="QLabel" name="totalTitle">
       <property name="text">
        <string>Total:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="totalValue">
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="receiptLayout">
     <item>
      <widget class="QPushButton" name="cancelsaleBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="onholdBtn">
       <property name="text">
        <string>On Hold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewholdBtn">
       <property name="text">
        <string>View Holds</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>